frontend/src/Student/pages/Potfolio.js

```diff
@@ -8,7 +8,6 @@ const Portfolio = () => (
     <Row>
       <Col>
         <Portfolios />
-        {/* TODO: Show all open shows w/ link to any submissions a student made to them */}
```

Comment:
@JohnPMurray I actually don't know why this is here. Was this your addition?

Reply:
nope